app/controllers/application_controller.rb

```diff
@@ -87,4 +89,8 @@ def set_authenticated_user_header
       GdsApi::GovukHeaders.set_header(:x_govuk_authenticated_user, current_user.uid)
     end
   end
+
+  def notify_bad_request(_exception)
+    render plain: "Error: One or more recipients not in GOV.UK Notify team (code: 400)", status: :bad_request
```

Comment:
It would probably be good to check the exception before giving such a specific message, as there's probably other things that can lead to a bad request I'm guessing?

Reply:
Probably worth writing a test for it too 🤔

Reply:
> there's probably other things that can lead to a bad request I'm guessing?

If I'm reading [their documentation](https://docs.notifications.service.gov.uk/ruby.html#error-codes) correctly, the only other thing that would raise a `BadRequestError` is if the service was in trial mode and couldn't send messages at all. I'm not actually sure in what situation that would occur that wasn't already covered by the not-in-team error, since rate limiting is a separate error code.

There are other reasons a `BadRequestError` might occur but they mostly relate to sending files etc, or on other API calls like retrieving a template, so they don't apply here — on the other hand, perhaps if there's even the possibility we might use file attachments in future it's better to be careful with this.